frontend/ui/data/creoptions.lua

```diff
@@ -437,7 +437,7 @@ Note that your selected font size is not affected by this setting.]]),
                 item_text = not Device:isTouchDevice() and {_("decrease"), _("increase")} or nil,
                 more_options = true,
                 more_options_param = {
-                    value_min = 7,
+                    value_min = 12,
```

Comment:
Btw, this one I'm not really familiar with but could it also be UI vs engine?

Reply:
There is
https://github.com/koreader/koreader/blob/827581ac728330b38853450ff59c3afca00fe488/frontend/apps/reader/modules/readerfont.lua#L252-L254

Reply:
Had a hard time understanding why changing these 12 to 7 didn't allow for < 12.
We would need to also change it here:
https://github.com/koreader/koreader-base/blob/14bb24f8bb29005f2aee091d63da33c2fb0db142/cre.cpp#L1797-L1799

Anyway, 12 is small enough (and I'm off tomorrow for a dozen of days, so not making last minutes PRs :)

Reply:
Also, this cre.cpp is a limit to the font size after it is scaleBySized - so if we allow readerfont to accept 7, it might be scaled to > 12 and work. (while on my small emulator, no scaling, so the min everywhere is 12).
But then, we set limits on the unscaled font size - and its scaling is variable, so we may always have a difference between the min values enforced...

Reply:
I'd be wary changing this given the scaleBySize involvement. The normal minimum DPI of 160 would probably mean that 7 is 12.

Reply:
@zwim : remember why ?

Reply:
In that case reverting it sounds safe. :+1: